pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -265763,7 +265763,6 @@ self: {
        license = lib.licenses.bsd3;
        hydraPlatforms = lib.platforms.none;
```

Comment:
```suggestion
```

You also need to remove `hydraPlatforms = none`, otherwise Hydra won't build it and it won't end up in the cache!

Reply:
Hmm, I don't appear to be able to auto-commit this through the GitHub UI for some reason...

@sorki Could you commit this change as well?

Reply:
Sure. Good to know and thanks!